src/caret_analyze/record/records_service/response_time.py

```diff
@@ -509,8 +506,6 @@ def __init__(
         response_map = ResponseMap(records, columns)
         self._response_map_all = ResponseMapAll(records, columns)
         self._records = ResponseRecords(response_map)
-        self._timeseries = ResponseTimeseries(self._records)
```

Comment:
In the following two functions in the ResponseTime class, this deleted part is used:
- `to_best_case_records()`
- `to_worst_case_records`